utils.go

```diff
@@ -3,18 +3,17 @@ package redisgraph
 import (
 	"crypto/rand"
 	"fmt"
-	"reflect"
 	"strings"
 )
 
 // go array to string is [1 2 3] for [1, 2, 3] array
 // cypher expects comma separated array
 func arrayToString(arr interface{}) interface{} {
```

Comment:
Shouldn't this be
`func arrayToString(arr []interface{}) interface{} {`

Reply:
changed